PowerEditor/src/WinControls/AnsiCharPanel/ansiCharPanel.cpp

```diff
@@ -140,7 +140,7 @@ void AnsiCharPanel::insertChar(unsigned char char2insert) const
 	}
 	else
 	{
-		MultiByteToWideChar(codepage, 0, charStr, -1, wCharStr, sizeof(wCharStr));
+		MultiByteToWideChar(codepage, 0, charStr, -1, wCharStr, _countof(wCharStr));

```

Comment:
This one has been omitted by the static analysis, but still corrected it for the sake of completeness.